packages/mongodb/src/MongoDriver.ts

```diff
@@ -122,6 +122,17 @@ export class MongoDriver extends DatabaseDriver<MongoConnection> {
     await this.rethrow(Promise.all(promises));
   }
 
+  async refreshCollections(options: RefreshCollectionsOptions = {}): Promise<void> {
+    options.ensureIndexes ??= true;
+
+    await this.rethrow(this.dropCollections());
+    await this.rethrow(this.createCollections());
+
+    if (options.ensureIndexes) {
+      await this.rethrow(this.ensureIndexes());
```

Comment:
no need to use `rethrow` from this method - all the methods it calls are using it internally
```suggestion
    await this.dropCollections();
    await this.createCollections();

    if (options.ensureIndexes) {
      await this.ensureIndexes();
```

Reply:
removed `rethrow`